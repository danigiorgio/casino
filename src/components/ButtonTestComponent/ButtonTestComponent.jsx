import React from 'react';

function ButtonTestComponent({ children }) {
  return <div>{children}</div>;
}

export default ButtonTestComponent;
