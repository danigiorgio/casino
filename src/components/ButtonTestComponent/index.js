import ButtonTestComponent from './ButtonTestComponent';

export default ButtonTestComponent;
