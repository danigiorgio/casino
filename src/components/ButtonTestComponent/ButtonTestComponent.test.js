import React from 'react';
import { render } from '@testing-library/react';

import ButtonTestComponent from './ButtonTestComponent';

describe('<ButtonTestComponent />', () => {
  it('should render ButtonTestComponent', () => {
    const { container } = render(<ButtonTestComponent />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
