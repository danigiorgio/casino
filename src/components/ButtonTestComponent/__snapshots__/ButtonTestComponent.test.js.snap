// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonTestComponent /> should render ButtonTestComponent 1`] = `<div />`;
